cvat-ui/src/components/task-page/job-list.tsx

```diff
@@ -24,22 +27,45 @@ const baseURL = core.config.backendAPI.slice(0, -7);
 interface Props {
     taskInstance: any;
     registeredUsers: any[];
+    currentJobId: number | null;
     onJobUpdate(jobInstance: any): void;
+    onJobremove(): void;
 }
 
-export default function JobListComponent(props: Props): JSX.Element {
+function JobListComponent(props: Props & RouteComponentProps): JSX.Element {
     const {
         taskInstance,
         registeredUsers,
         onJobUpdate,
+        onJobremove,
+        currentJobId,
+        history: {
+            push,
+        },
     } = props;
 
-    const { jobs } = taskInstance;
+    const { jobs, id: taskId } = taskInstance;
     const columns = [{
         title: 'Job',
         dataIndex: 'job',
         key: 'job',
-        render: (id: number): JSX.Element => (<a href={`${baseURL}/?id=${id}`}>{ `Job #${id}` }</a>),
+        render: (id: number): JSX.Element => (
+            <div>
+                <Button
+                    type='link'
+                    onClick={(): void => {
+                        if (currentJobId !== id) {
+                            onJobremove();
+                        }
```

Comment:
Probably it would better if implement it in ``getJobAsync`` function.
Just dispatch one more action if another task is already opened. 
It would look much easier.

Reply:
One more option which looks good is to use ReactRouter::history::listen to close a previous job.